app/src/components/slider/Slider.tsx

```diff
@@ -0,0 +1,248 @@
+        var(--ac-global-rounding-medium)
+      );
+      z-index: -1;
+    }
+
+    &[data-dragging] {
+      background: white;
+    }
+
+    &[data-focus-visible] {
+      outline: 2px solid var(--ac-focus-ring-color);
+    }
+  }
+
+  &[data-orientation="horizontal"] {
+    flex-direction: column;
+    width: 100%;
+
+    .ac-slider-track {
+      height: var(--ac-slider-track-height, var(--ac-global-border-size-thick));
+      width: 100%;
+
+      /* background track line */
+      &:before {
+        width: 100%;
+        top: 50%;
+        transform: translateY(-50%);
+      }
+
+      /* filled track line */
+      &:after {
+        left: var(--slider-start);
+        width: calc(var(--slider-end) - var(--slider-start));
+        top: 50%;
+        transform: translateY(-50%);
+        z-index: 1;
+      }
+    }
+
+    .ac-slider-thumb {
+      top: 50%;
+      z-index: 2;
+    }
+  }
+`;
+
+export type SliderProps<T> = AriaSliderProps<T> &
+  StylableProps &
+  React.PropsWithChildren<{
+    label?: string;
+    thumbLabels?: string[];
+  }>;
+
+export function Slider<T extends number | number[]>({
+  label,
+  thumbLabels,
+  children,
+  css: _css,
+  ...props
+}: SliderProps<T>) {
+  return (
+    <AriaSlider css={css(sliderCSS, _css)} {...props}>
+      {label && <Label className="ac-slider-label">{label}</Label>}
+      <AriaSliderOutput className="ac-slider-output">
+        {typeof children === "undefined" ? <SliderTextField /> : children}
+      </AriaSliderOutput>
+      <AriaSliderTrack
+        className="ac-slider-track"
+        style={({ state }) => {
+          // check state to determine how we should fill the track
+          // generate css vars for single thumb
+          if (state.values.length === 1) {
+            return {
+              "--slider-start": "0%",
+              "--slider-end": `${state.getThumbPercent(0) * 100}%`,
+            } as React.CSSProperties;
+          }
+
+          // generate css vars for multi-thumb
+          return {
+            "--slider-start": `${state.getThumbPercent(0) * 100}%`,
+            "--slider-end": `${state.getThumbPercent(1) * 100}%`,
+          } as React.CSSProperties;
+        }}
+      >
+        {({ state }) => (
+          <>
+            {state.values.map((_, i) => (
+              <AriaSliderThumb
+                key={i}
+                index={i}
+                aria-label={thumbLabels?.[i]}
+                className="ac-slider-thumb"
+              />
+            ))}
+          </>
+        )}
+      </AriaSliderTrack>
+    </AriaSlider>
+  );
+}
+
+export function SliderNumberField({
+  value: _value,
+  onChange: _onChange,
+  ...props
+}: NumberFieldProps) {
+  const { step, getThumbMinValue, getThumbMaxValue, values, setThumbValue } =
+    useContext(SliderStateContext)!;
+  const value = _value ?? values[0].toString() ?? "";
+  const labelProps = useSlottedContext(LabelContext)!;
+  return (
+    <NumberField
```

Comment:
<img width="374" alt="Screenshot 2025-03-06 at 3 21 05 PM" src="https://github.com/user-attachments/assets/de1f2163-561b-4beb-9a72-9dbf15325b22" />
Can we make the max width a bit smaller? Feels kinda chonky 🤣

Reply:
Yes good call